data/sites.json

```diff
@@ -278,32 +290,85 @@
+                "city": "Gardner",
+                "zip": "01440"
+            },
             {
                 "street": "68 South Main St",
                 "city": "Haverhill",
                 "zip": "01835"
             },
-            { "street": "1145 Main St", "city": "Holden", "zip": "01520" },
-            { "street": "520 W. Main St", "city": "Hyannis", "zip": "02601" },
-            { "street": "25 Park St", "city": "Lee", "zip": "01238" },
-            { "street": "21 South St", "city": "Mashpee", "zip": "02649" },
-            { "street": "825 Morton St", "city": "Boston", "zip": "02126" },
-            { "street": "90 River St", "city": "Mattapan", "zip": "02126" },
+            {
+                "street": "1145 Main St",
+                "city": "Holden",
+                "zip": "01520"
+            },
+            {
+                "street": "520 W. Main St",
+                "city": "Hyannis",
+                "zip": "02601"
+            },
+            {
+                "street": "25 Park St",
+                "city": "Lee",
+                "zip": "01238"
+            },
+            {
+                "street": "21 South St",
+                "city": "Mashpee",
+                "zip": "02649"
+            },
+            {
+                "street": "825 Morton St",
+                "city": "Boston",
+                "zip": "02126"
+            },
+            {
+                "street": "90 River St",
+                "city": "Mattapan",
+                "zip": "02126"
+            },
             {
                 "street": "37 Cheshire Road",
                 "city": "Pittsfield",
                 "zip": "01201"
             },
-            { "street": "430 Broadway", "city": "Revere", "zip": "02151" },
+            {
+                "street": "430 Broadway",
+                "city": "Revere",
+                "zip": "02151"
+            },
             {
                 "street": "1890 Columbus Ave",
                 "city": "Roxbury",
                 "zip": "02119"
             },
-            { "street": "416 Warren St", "city": "Roxbury", "zip": "02119" },
-            { "street": "166 Walnut St", "city": "Saugus", "zip": "01906" },
-            { "street": "296 Buffinton St", "city": "Somerset", "zip": "02726" }
+            {
+                "street": "416 Warren St",
+                "city": "Roxbury",
+                "zip": "02119"
+            },
+            {
+                "street": "166 Walnut St",
+                "city": "Saugus",
+                "zip": "01906"
+            },
+            {
+                "street": "296 Buffinton St",
+                "city": "Somerset",
+                "zip": "02726"
+            }
         ]
+    },
+    "HeywoodHealthcare": {
+        "name": "Heywood Healthcare",
+        "street": "171 KENDALL POND W",
+        "city": "Gardner",
+        "state": "MA",
+        "zip": "01440",
+        "website": "https://www.heywood.org/covid-19/covid19-vaccine",
```

Comment:
We don't use this "website" key anywhere - mind taking it out?

Reply:
I'll replace with signUpLink as indicated in your comments for #102.